Not overwriting project

henpe071 opened the pull request:
When creating a new project, before the prompt accepts the answer, it checks if the path already exists. If it does, it alerts the user and asks if he/she wanna continue and overwrite the old one, open the old one or just cancel the action.

henpe071 committed:
duplication check för creating new projects

You will now get a popup asking for confirmation if you try to create a new project when there already exist one with that name.

henpe071 committed:
Now if the project already exists you can open it from the prompt

henpe071 committed:
Added some comments and removed some prints

fredrikiselius commented on ViAn/GUI/projectdialog.cpp:
```diff
@@ -57,9 +59,33 @@ void ProjectDialog::browse_btn_clicked() {
+        // Create confirmation dialog since the path already exists
+        QMessageBox msg_box;
+        msg_box.setModal(true);
+        msg_box.setText("This folder already exist. Are you sure you wan't to continue?");
```
want

fredrikiselius commented on ViAn/GUI/projectdialog.cpp:
```diff
@@ -57,9 +59,33 @@ void ProjectDialog::browse_btn_clicked() {
+                m_dir = "C:" + m_dir;
+            }
+            emit open_project(m_dir + name_text->text());
+            qDebug() << m_dir;
```
Remove debug messages

fredrikiselius reviewed (commented):
When a project is overwritten the treestate from the old project is still saved. It needs to be removed else placeholder items will appear in the project tree when loading.

henpe071 committed:
Fixed some comments

henpe071 commented:
I fixed the comment and removed the debug messages. I will wait with the problem that the project still partly remains after it's been overwritten. That problem should be easily fixed once we get the temporary save files. So instead of doing a work-around here to fix it I'll just wait.
